doc/ChangeLog

```diff
@@ -11,11 +11,11 @@ Resolved issues:
 
 ESCOMP/ctsm#683 :  Writing out FATES parameters only on masterproc.
 ESCOMP/ctsm#874 :  Changing the number of timesteps to run for
-				   SMS_P720x1_Ln3.hcru_hcru.I2000Clm50BgcCruGs.cheyenne_intel.clm-coldStar
-				   to prevent failing with CMEPS.
+                   SMS_P720x1_Ln3.hcru_hcru.I2000Clm50BgcCruGs.cheyenne_intel.clm-coldStar
```

Comment:
Changed tabs to spaces because this didn't show up aligned in my editor with tabs.